frontend/src/components/auth/teacher-signup/steps/TeacherSignUpOne.tsx

```diff
@@ -137,19 +139,9 @@ const TeacherSignupOne = ({
         <FormLabel color="grey.400">What grades do you teach?</FormLabel>
         <CheckboxGroup onChange={onChangeGrades} value={gradesValues}>
           <SimpleGrid columns={3}>
-            {[
-              "K",
-              "Grade 3",
-              "Grade 6",
-              "Grade 1",
-              "Grade 4",
-              "Grade 7",
-              "Grade 2",
-              "Grade 5",
-              "Grade 8",
-            ].map((grade) => (
+            {Object.keys(Grade).map((grade) => (
```

Comment:
i think you can also use the constant i mentioned below to map values to Checkbox

Reply:
Done!